integrations/adobe-analytics/lib/heartbeat.js

```diff
@@ -0,0 +1,180 @@
+
+var heartbeats = {
+
+  extractMediaMetadata: function (track) {
```

Comment:
Please add [jsdoc](https://jsdoc.app/tags-returns.html) to document:
- `@returns`
- `@param`
- brief summary of function intent

Reply:
Probably unnecessary for a lot of `chromecast...` functions e.g. `chromecastHeartbeatVideoStart` but would be nice on the more custom / less self-explanatory functions.

Reply:
Added jsdoc for the only helper method in that file extractMediaMetadata